CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,69 @@
+# Code of Conduct
+
+## Our Pledge
```

Comment:
This plege is for a community project repository, not for an individual owned one. Rewrite it or remove it.